package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nosleep.js",
-  "version": "0.9.0",
+  "version": "0.10.0",
```

Comment:
Would you be able to remove this version bump from the PR? 

I plan to then merge this PR with a version bump happening in a separate (collective) way.